tests/ct1_integration/Attendees_TableTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Tribe\Tickets;
+
+use TEC\Events\Custom_Tables\V1\Models\Occurrence;
+use TEC\Events\Custom_Tables\V1\Models\Occurrence as Occurrence_Model;
+use Tribe\Tickets\Test\Commerce\Attendee_Maker;
+use Tribe\Tickets\Test\Commerce\PayPal\Ticket_Maker as PayPal_Ticket_Maker;
+use Tribe\Tickets\Test\Commerce\RSVP\Ticket_Maker as RSVP_Ticket_Maker;
+use Tribe\Tickets\Test\Traits\CT1\CT1_Fixtures;
+use Tribe__Tickets__Attendees;
+use Tribe__Tickets__Attendees_Table as Attendees_Table;
+use Tribe__Tickets__Data_API as Data_API;
+
+class Attendees_TableTest extends \Codeception\TestCase\WPTestCase {
+	use CT1_Fixtures;
+	use RSVP_Ticket_Maker;
+	use PayPal_Ticket_Maker;
+	use Attendee_Maker;
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		// Enable post as ticket type.
+		add_filter( 'tribe_tickets_post_types', function () {
+			return [ 'post' ];
+		} );
+
+		// Enable Tribe Commerce.
+		add_filter( 'tribe_tickets_commerce_paypal_is_active', '__return_true' );
+		add_filter( 'tribe_tickets_get_modules', function ( $modules ) {
+			$modules['Tribe__Tickets__Commerce__PayPal__Main'] = tribe( 'tickets.commerce.paypal' )->plugin_name;
+
+			return $modules;
+		} );
```

Comment:
Tribe Commerce is deprecated, we should add the tests for Tickets Commerce instead. @bordoni what do you think?

Reply:
Agree with @rafsuntaskin here.